### ISSUE TITLE
[bash] abort exection on error

### ISSUE THREAD

**Issue by carlwgeorge** [iuscommunity/automation-examples#5]
There is a minor problem with the enable-ius.sh script in the bash branch.  If the ius-release or epel-release RPM URLs fail to resolve for whatever reason, the script keeps running and tries to import a key file that doesn't exist, resulting in a misleading error.

```
# rpm --import /etc/pki/rpm-gpg/IUS-COMMUNITY-GPG-KEY
error: /etc/pki/rpm-gpg/IUS-COMMUNITY-GPG-KEY: import read failed(2).
```

Using `set -e` (abort for any non-zero exit code) at the start of the script should be able to handle this, as long as yum properly produces a non-zero exit code when `yum install <URL_TO_RPM>` fails to resolve the given URL.

**Comment by T1c0MmRtODFORlpt** [comment]
+1

**Comment by carlwgeorge** [comment]
This was fixed in 5a50c106cd48465b05e6f438a60731c306a84bb0.
